src/Components/WeatherDetails/WeatherDetails.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import './WeatherDetails.scss';
+
+const WeatherDetails = ({ apiRequestData }) => {
+  const { cod } = apiRequestData;
+  const locationNotFound = cod !== 200 ? true : false;
+
+  if (locationNotFound) {
+    return (
+      <div className="weather-details">
+        <div className="weather-details__message alert alert-danger">
+          Sorry, we didn't find the location
+        </div>
+      </div>
+    );
+  }
+
+  const {
+    name,
+    main: { temp, temp_max, temp_min, humidity, pressure },
+    wind: { speed },
+    sys: { country },
+  } = apiRequestData;
+
+  return (
+    <div className="weather-details">
+      <div className="weather-details__message alert alert-success">
+        Weather data for the location: <b>{name}</b>
+      </div>
+      <div className="weather-details__container">
+        <div className="weather-details__icon"></div>
```

Comment:
WTF is that?

Reply:
Schould I split it on components I'm right?

Reply:
No you should put Icon as a component here or Image 

`<WeatherIcon />  `

and inside weather icon.jsx

`const WeatherIcon = () => (<img src=/..... ); `

Reply:
I know my mate but it's under construction, I'll put icon but I neet to prepare them :D